modules/get-started/pages/release-notes/redpanda.adoc

```diff
@@ -41,8 +41,68 @@ Redpanda Schema Registry now supports an import mode that allows you to import e
 Starting with this release, import mode must be used when importing schemas. Read-write mode no longer allows specifying a schema ID and version when registering a schema.
 See xref:manage:schema-reg/schema-reg-api.adoc#set-schema-registry-mode[Use the Schema Registry API] for more information.
 
-== Deprecations
+== New configuration properties
 
-Several TLSv1.2 and TLSv1.3 cipher suites have been deprecated. See xref:upgrade:deprecated/index.adoc[Deprecated Features].
+Redpanda 25.3 introduces several new configuration properties:
+
+**Shadowing:**
+
+* xref:reference:properties/cluster-properties.adoc#enable_shadow_linking[`enable_shadow_linking`]: Enable creating Shadow Links from this cluster to a remote source cluster (Enterprise license required).
+
+**Timestamp validation:**
+
+These properties replace the deprecated `log_message_timestamp_alert_after_ms` and `log_message_timestamp_alert_before_ms` properties, providing enhanced control over timestamp validation. See xref:develop:produce-data/configure-producers.adoc[Configure Producers] for more information.
+
+* xref:reference:properties/cluster-properties.adoc#log_message_timestamp_after_max_ms[`log_message_timestamp_after_max_ms`]: Maximum allowable timestamp difference for records with timestamps later than the broker's timestamp.
+* xref:reference:properties/cluster-properties.adoc#log_message_timestamp_before_max_ms[`log_message_timestamp_before_max_ms`]: Maximum allowable timestamp difference for records with timestamps earlier than the broker's timestamp.
+* xref:reference:properties/topic-properties.adoc#messagetimestampaftermaxms[`message.timestamp.after.max.ms`]: Topic-level override for maximum timestamp difference (future).
+* xref:reference:properties/topic-properties.adoc#messagetimestampbeforemaxms[`message.timestamp.before.max.ms`]: Topic-level override for maximum timestamp difference (past).
+
+**Audit logging:**
+
+* xref:reference:properties/cluster-properties.adoc#audit_use_rpc[`audit_use_rpc`]: Produce audit log messages using internal Redpanda RPCs. See xref:manage:audit-logging.adoc[] for more information.
+
+**Object storage:**
+
+* xref:reference:properties/object-storage-properties.adoc#cloud_storage_client_lease_timeout_ms[`cloud_storage_client_lease_timeout_ms`]: Maximum time to hold a connection to object storage before closing it.
+* xref:reference:properties/object-storage-properties.adoc#cloud_storage_cluster_name[`cloud_storage_cluster_name`]: Optional unique name to disambiguate cluster metadata in object storage for xref:manage:whole-cluster-restore.adoc[Whole Cluster Restore].
```

Comment:
@nvartolomei did we want to hide this from the public release notes? I thought this was for serverless only

Reply:
I'd omit it from release notes. Fine to be documented otherwise as our internal teams are also consumers of the same docs.

Reply:
Added a note to say it's internal only as well.